src/border_agent/border_agent.cpp

```diff
@@ -130,7 +136,9 @@ struct StateBitmap
         bitmap |= mAvailability << 5;
         bitmap |= mBbrIsActive << 7;
         bitmap |= mBbrIsPrimary << 8;
-
+#if OTBR_EPSKC_ENABLE
+        bitmap |= mEPSKcSupported << 11;
+#endif
```

Comment:
```suggestion
        bitmap |= mEPSKcSupported << 11;
```
Similar to above comments.

Reply:
thanks! Accept.